azurelinuxagent/ga/update.py

```diff
@@ -332,22 +332,22 @@ def run(self, debug=False):
 
                     utc_start = datetime.utcnow()
 
-                    last_etag = exthandlers_handler.last_etag
                     exthandlers_handler.run()
 
                     remote_access_handler.run()
 
-                    if last_etag != exthandlers_handler.last_etag:
+                    if exthandlers_handler.changed():
                         self._ensure_readonly_files()
                         duration = elapsed_milliseconds(utc_start)
-                        logger.info('ProcessGoalState completed [incarnation {0}; {1} ms]',
-                                    exthandlers_handler.last_etag,
+                        description = exthandlers_handler.goal_state_description()
+                        logger.info('ProcessGoalState completed [{0}; {1} ms]',
```

Comment:
The message reads like this

> ProcessGoalState completed [FastTrack: SeqNo=2, Reason=LastFT: FastTrack=NoChange, Fabric=NoChange; 2117 ms]

The colons seem a little bit out of place

Reply:
Removed